### PR TITLE
remove references to varnishcheck

### PR DESCRIPTION
**Pull request by EdwardAngert**
Closes: #5864 

## Summary

**[Testing Global CDN Caching](https://pantheon.io/docs/test-global-cdn-caching)** - Varnishcheck is retiring and has been removed from the docs. [PR 5867](https://github.com/pantheon-systems/documentation/pull/5867)

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-5867](https://pr-5867--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `67` | `54` | `-13` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `100` | `98` | `-2` |

Lighthouse Audit Reports:

- [`pr-5867` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/22511/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/22511/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-5867--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by EdwardAngert** [comment]
@ari-gold check out the [new order in the preview](https://pr-5867--pantheon-docs.my.pantheonfrontend.website/docs/test-global-cdn-caching)
once you 👍  we'll move it to the next review on Docs

**Comment by ari-gold** [comment]
This section "How to Read HTTP Headers" is quite long. Can we move it to the bottom so people can see instructions on how to use Chrome, Mozilla, etc to see the Age header?  Also, what do you think about 

changing 
```
curl -I -H "accept-encoding: gzip, deflate, br" https://scalewp.io
HTTP/2 200
cache-control: public, max-age=86400
content-encoding: gzip
content-type: text/html; charset=UTF-8
link: <https://scalewp.io/wp-json/>; rel="https://api.w.org/"
link: <https://scalewp.io/>; rel=shortlink
server: nginx
x-pantheon-styx-hostname: styx-fe3-a-906849904-7zhv4
x-styx-req-id: styx-460041beb0cbd966edfdeac5f09e8c50
via: 1.1 varnish
accept-ranges: bytes
date: Mon, 16 Apr 2018 16:30:18 GMT
via: 1.1 varnish
age: 44742
x-served-by: cache-mdw17344-MDW, cache-jfk8146-JFK
x-cache: HIT, HIT
x-cache-hits: 1, 1
x-timer: S1523896219.500596,VS0,VE1
vary: Accept-Encoding, Cookie, Cookie
content-length: 41369
```

to something shorter like:

```
curl -I -H "accept-encoding: gzip, deflate, br" https://scalewp.io
HTTP/2 200
cache-control: public, max-age=86400
<snip>
age: 44742
<snip>
```

File: source/content/test-global-cdn-caching.md [hunk 1]
```diff
@@ -4,47 +4,8 @@ description: Detailed information on how to determine if CDN caching is working
 categories: [performance]
 tags: [cache, cdn]
 ---
-## Verify the Global CDN is Working on Your Pantheon Site
 
-Use the [web utility](https://varnishcheck.pantheon.io/) to check to see if caching is working on your Pantheon-hosted website. This tool performs up to two web requests to your site and will check the headers to determine if the CDN can cache your site. If not, it will make recommendations specific to your site configuration. Please note that this utility does not check for cookies that are set in your frontend code (i.e. JavaScript). If you have any feedback, let us know by [contacting support](/support).
-
-## Test If Global CDN Caching Is Working by Reading HTTP Headers
```
Comment (alexfornuto) [inline source/content/test-global-cdn-caching.md:11]: This information is still correct, even if you get it from `curl`, no?

**Comment by EdwardAngert** [comment]
👍  @ari-gold , I think the latest commit resolves both of your suggestions, so I'm moving this to copy review. Leaving you tagged as a reviewer for the day for some 👀

**Pull request merged**